src/cloud/components/SeriesCardinalityIncreaseForm.tsx

```diff
@@ -4,38 +4,22 @@ import React, {FC} from 'react'
 // Components
 import {
   Form,
-  Button,
   Heading,
-  ButtonShape,
   InfluxColors,
-  ComponentSize,
-  ComponentColor,
   HeadingElement,
 } from '@influxdata/clockface'
 
 export const SeriesCardinalityIncreaseForm: FC = () => {
-  const handleSubmit = (): void => {
-    window.open(`https://support.influxdata.com/s/login`)
-  }
-
   return (
     <div>
       <Form.Divider lineColor={InfluxColors.Grey15} />
       <Heading element={HeadingElement.H4}>
         Request Series Cardinality Limit Increase
       </Heading>
       <p>
-        To request a series cardinality limit increase, please contact our
-        support team.
+        To request a cardinality limit increase, please use the Contact Support
```

Comment:
We should put "Contact Support" in quotes. Same with "Help." This can be applied throughout.